web/styles/settings.css

```diff
@@ -1886,6 +1886,23 @@ $option_title_width: 180px;
         display: inline-block;
     }
 
+    #user_filters {
+        display: flex;
+        float: right;
+        flex-wrap: wrap;
+
+        #select_role_widget {
+            /* The font size here is 0.1 em less than input.search which
+            is just beside this widget. We kept it less so that their
+            resultant height is same in the end. This widget was slightly
+            bigger otherwise. */
+            font-size: 0.9em;
```

Comment:
I don't like modifying font-size to adjust height of the dropdown widget here. You can override styles for `.dropdown-widget-button` to adjust the height.

Reply:
Fixed it